src/lets-version.js

```diff
@@ -532,6 +573,43 @@ export async function applyRecommendedBumpsByPackage(
         } else await fs.writeFile(c.changelogPath, `${changelogUpdates}${existingChangelog}`, 'utf-8');
       }),
     );
+
+    if (rollupChangelog) {
+      // User wants an aggregated changelog at the root.
+      // if this repo only has a single package AND that package is marked
+      // as the root package, do nothing
+
+      const continueWithRollupChangelog =
+        (synchronized.packages.length === 1 && !synchronized.packages[0]?.root) || synchronized.packages.length > 1;
+
+      if (continueWithRollupChangelog) {
+        const changelogUpdates = new ChangelogAggregateUpdate(fixedCWD, getFormattedChangelogDate(), changelogInfo);
+        if (!dryRun) await fs.ensureFile(changelogUpdates.changelogPath);
+
+        let existingChangelog = '';
+        try {
+          existingChangelog = await fs.readFile(changelogUpdates.changelogPath, 'utf-8');
+        } catch (error) {}
+
+        const updatesToWrite = customConfig?.changelog?.changeLogRollupFormatter
+          ? customConfig?.changelog?.changeLogRollupFormatter(changelogUpdates)
+          : changelogUpdates.toString();
+
+        if (dryRun) {
+          console.info(
+            `Will write the following rollup changelog updated to ${changelogUpdates.changelogPath}:${os.EOL}${os.EOL}${
+              updatesToWrite || ''
+            }`,
+          );
+        } else if (updatesToWrite) {
+          await fs.writeFile(
+            changelogUpdates.changelogPath,
+            `${changelogUpdates.toString()}${existingChangelog}`,
```

Comment:
Should this be `${updatesToWrite}${existingChangelog}`?

Reply:
Addressed in db0bd6830e62fc4b5548c35ea194d9691fe2b318